### ISSUE TITLE
How to get a node by property without its labels

### ISSUE THREAD

**Issue by DachuanZhao** [neo4j-contrib/neomodel#396]
Here is the code
```
class Person(StructuredNode):
    __abstract_node__ = True

    uid = UniqueIdProperty()
    name = StringProperty(unique_index=True)
    age = IntegerProperty(index=True, default=0)

class Male(Person):
    pass

class Female(Person):
    pass
```
I want to find a person named "ANDY" but I don't konw it's a man or a woman.Now my solution is
`Male.nodes.get_or_none(name=name) or Female.nodes.get_or_none(name=name)`
Is there a better way to solve it?

**Comment by aanastasiou** [comment]
@DachuanZhao Not necessarily, you can use plain simple CYPHER for that, even through neomodel.

neomodel brings with it a specific type of thinking called [Model Driven Engineering](https://en.wikipedia.org/wiki/Model-driven_engineering). There, the "Model" is the main artifact. You don't really care how things are going to end up in the database and in some cases you don't even care what database your data is going to end up in because the "platform-specific" layer could simply change by configuration.

The point here is that you can express the same model over a relational or graph data model (and enjoy different benefits in each case) without caring about the specifics of the implementation.

So, from that point of view, the "labels" that you perceive as a problem are actually fundamental to the building of the model.

You start FROM laying out your data model and the queries write themselves.

This is why I asked you why have you chosen to model your data in that way.
